docs/WritingTests.asciidoc

```diff
@@ -346,34 +346,87 @@ In short, writing multi-machine tests adds a few more layers of complexity:
 3. actual technical realization (i.e. <<Networking.asciidoc#networking,custom networking>>)
 
 ==== Job dependencies
-There are different dependency *types* (see subsequent section). Additionally, dependencies can be machine-specific (see "Inter-machine dependencies" section).
+There are three dependency *types*: `CHAINED`, `DIRECTLY_CHAINED` and `PARALLEL`
 
-===== Dependency types
-There are 3 types of dependencies: `CHAINED`, `DIRECTLY_CHAINED` and `PARALLEL`
+A dependency is always between two jobs where one of the jobs is the _parent_
+and one the _child_. The concept of parent and child jobs is *orthogonal* to
```

Comment:
I consider this sentence "The concept of parent and child jobs is *orthogonal*" hard to read. Maybe we should try to move some explanation further down so that the difference for "chained" and "parallel" appears earlier?

Reply:
I agree that the meaning of *orthogonal* in this context is hard to understand. Maybe "The concept of parent and child jobs is independent of …". Note that I've been adding that sentence because people seem to mix it up. Therefore I thought before going into specific dependency types it makes sense to clear it up.

Reply:
ok, independant is good. But also I suggested to simply move this explanation further down.

Reply:
Or maybe "The concept of parent and child jobs applies regardless of the dependency type."?

Reply:
Now I am again confused. Isn't child+parent defined by using the CHAINED dependency?

Reply:
So after talking about we realized that we use parent+child and people sometimes wrongly assume that childs always run *after* parents when what we have is that parents more "lead the way" and the dependency type configures how, if parallel or chained